src/components/apiManager.js

```diff
@@ -0,0 +1,11 @@
+const spaceXAPI = {
+  getRocketsData() {
+    const rockets = fetch('https://api.spacexdata.com/v3/rockets')
+      .then((rawResponse) => rawResponse.json())
+      .then((response) => response);
+
+    return rockets;
+  },
+};
```

Comment:
**Optional Change**
- [ ] Let us try to use the `async await` syntactic sugar for promises. It just makes the code more readable than chaining using the `.then` keyword